File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/constant/ViewProperties.java
package com.github.liulus.yurt.system.constant;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * @author liulu
 * @version V1.0
 * @since 2020/11/24
 */
@Data
@ConfigurationProperties(prefix = "yurt.view")
public class ViewProperties {

    private String cdnHost;

    private String ui;

}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/context/ViewProperties.kt
package com.github.liulus.yurt.system.context

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * @author liulu
 * @version V1.0
 * @since 2020/11/24
 */
@ConfigurationProperties(prefix = "yurt.view")
data class ViewProperties(
        var cdnHost: String? = null,
        var ui: String? = null,
)